test/e2e/config/docker.yaml

```diff
@@ -70,25 +70,31 @@ providers:
   - sourcePath: "../data/infrastructure-docker/cluster-template.yaml"
   - sourcePath: "../data/infrastructure-docker/cluster-template-mhc.yaml"
   - sourcePath: "../data/infrastructure-docker/cluster-template-kcp-adoption.yaml"
+  - sourcePath: "../data/infrastructure-docker/cluster-template-machine-pool.yaml"
 
 variables:
   KUBERNETES_VERSION: "v1.18.2"
   ETCD_VERSION_UPGRADE_TO: "3.4.3-0"
   COREDNS_VERSION_UPGRADE_TO: "1.6.7"
   KUBERNETES_VERSION_UPGRADE_TO: "v1.18.2"
   KUBERNETES_VERSION_UPGRADE_FROM: "v1.17.2"
+  MIN_NUMBER_OF_REPLICAS: "1"
+  MAX_NUMBER_OF_REPLICAS: "5"
   DOCKER_SERVICE_DOMAIN: "cluster.local"
   DOCKER_SERVICE_CIDRS: "10.128.0.0/12"
   # IMPORTANT! This values should match the one used by the CNI provider
   DOCKER_POD_CIDRS: "192.168.0.0/16"
   CNI: "./data/cni/kindnet/kindnet.yaml"
   EXP_CLUSTER_RESOURCE_SET: "true"
+  EXP_MACHINE_POOL: "true"
 
 intervals:
   default/wait-controllers: ["3m", "10s"]
   default/wait-cluster: ["5m", "10s"]
   default/wait-control-plane: ["10m", "10s"]
   default/wait-worker-nodes: ["5m", "10s"]
+  default/wait-machine-pool-nodes: ["5m", "10s"]
```

Comment:
is this a big enough timeout? last run failed with:
```
/home/prow/go/src/sigs.k8s.io/cluster-api/test/e2e/machine_pool.go:78
Timed out after 300.004s.
Expected
    <int>: 3
to equal
    <int>: 5
/home/prow/go/src/sigs.k8s.io/cluster-api/test/framework/machinepool_helpers.go:85
```

Reply:
I think with a reduced scale size (suggested above) we should be good.